packages/memory-storage/src/resource-clients/request-queue.ts

```diff
@@ -240,11 +240,10 @@ export class RequestQueueClient extends BaseClient implements storage.RequestQue
             throw new Error(`Request with ID ${id} not found in queue ${queue.name ?? queue.id}`);
         }
 
-        const currentTimestamp = Date.now();
-        const canProlong = (r: InternalRequest) => !r.orderNo || r.orderNo > currentTimestamp || r.orderNo < -currentTimestamp;
+        const canProlong = (r: InternalRequest) => !!r.orderNo;
```

Comment:
Why this change?

Reply:
Two reasons
1. Brings it more in line with the API, which strictly checks that the orderNo is present (and set to the value), and that the clientKey matches (which we aren't using in storages at this time but we probably need to support)
2. It allows locking a request that isn't locked yet (which I can't tell if the API supports, I think it doesn't due to said clientKey not matching)